hoot-services/pom.xml

```diff
@@ -136,7 +136,7 @@
                     parallelism, but haven't been able to get them to work...so forking instead. -->
                     <!-- parallel>methods</parallel>
                     <useUnlimitedThreads>true</useUnlimitedThreads -->
-                    <forkCount>4</forkCount>
+                    <forkCount>16</forkCount>
```

Comment:
I do see all of that is gone now.  No big deal, I'll revert the change.  But I run dual quad core hyperthread..."4" doesn't cut it :-)  I'll try to remember to not to commit my local change for this.